source-bigquery-batch/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG BASE_IMAGE=ghcr.io/estuary/base-image:v1
+
+# Build Stage
+################################################################################
+FROM golang:1.21-bullseye as builder
+
+WORKDIR /builder
+
+# Download & compile dependencies early. Doing this separately allows for layer
+# caching opportunities when no dependencies are updated.
+COPY go.* ./
+RUN go mod download
+
+COPY go                 ./go
+COPY source-boilerplate ./source-boilerplate
+
+RUN go install -v ./go/...
+RUN go install -v ./source-boilerplate/...
+RUN go install -v github.com/jackc/pgx/v4
```

Comment:
That's a dumb copy-paste oversight (twice now!). In the `source-postgres-batch` Dockerfile it serves a purpose (because getting that installed ahead of time means that connector rebuilds don't have to rebuild that branch of the dependency tree), and it's just been carried over by mistake. I'll remove that from here and the MySQL Dockerfile.